### PR TITLE
RHICOMPL-708 - Fix Compliance wording and % compliant

### PR DESCRIPTION
**Pull request by dLobatog**
## What

Before this, the card showed "passes" instead of "compliant" which is
the wording we use everywhere else in the app. Also, the percentage is
not computed properly without this fix, it's either 0 or 1.

### Before
Screenshot from 2020-06-23 15-00-30
Screenshot from 2020-06-23 14-56-48

### After
Screenshot from 2020-06-23 15-07-55
Screenshot from 2020-06-23 14-56-04

## Code Review
@christiemolloy @ryelo @AllenBW

## Design Review
@kybaker

### CODE REVIEW

File: src/SmartComponents/Compliance/ComplianceCard.js [hunk 1]
```diff
@@ -114,9 +114,11 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
                                                         </SplitItem>
                                                         <SplitItem>
                                                             {intl.formatMessage(messages.compliantScore,
-                                                                { score: Math.trunc(
-                                                                    policy.attributes.compliant_host_count / policy.attributes.total_host_count
-                                                                ) }
+                                                                { score: +Math.trunc(
```
Comment (dLobatog) [inline src/SmartComponents/Compliance/ComplianceCard.js:117]: `+` makes sure that the `.0` gets dropped, e.g if it's `100.0`, `50.0`, but not if it's any other number

Comment (akofink) [reply src/SmartComponents/Compliance/ComplianceCard.js:117]: I'm confused why we're using `Math.trunc` and `toFixed(1)`. `trunc` always returns an integer with no decimal, and `toFixed(1)` always returns the number with 1 decimal again - you're guaranteed to get a `.0` on the end. Am I missing something?

```js
> (Math.trunc(5/10 * 100)).toFixed(1)
'50.0'
> (Math.trunc(5/11 * 100)).toFixed(1)
'45.0'
> (+Math.trunc(5/10 * 100)).toFixed(1)
'50.0'
> (+Math.trunc(5/11 * 100)).toFixed(1)
'45.0'
> (5/10 * 100).toFixed(1)
'50.0'
> (5/11 * 100).toFixed(1)
'45.5'
```

Comment (dLobatog) [reply src/SmartComponents/Compliance/ComplianceCard.js:117]: Uh oh - removing it is the way to go indeed. 

```
+(5/11*100).toFixed(1)
45.5
+(5/10*100).toFixed(1)
50
```

Comment (akofink) [reply src/SmartComponents/Compliance/ComplianceCard.js]: Ok, much better :) This `+` trick is a bit of a hack; this code converts a number to a string and then back to a number again, but I'm cool with it :stuck_out_tongue: 

```js
> 5/11*100
45.45454545454545
> (5/11*100).toFixed(1)
'45.5'
> +(5/11*100).toFixed(1)
45.5
```

**Comment by AllenBW** [comment]
sorry for the merge conflict gents 😭 🦄

**Comment by AllenBW** [comment]
conflict resolved, work merged, look for it on beta soon

**Pull request merged**